src/components/editor/EditorArticleController.jsx

```diff
@@ -348,93 +356,107 @@ export default class EditorArticleController extends FalcorController {
         }
       }
 
+      // Dialog action button
+      const actions = [
+        <RaisedButton
+          label={changedStateMessage}
+          primary
+          style={styles.buttons}
+          type="submit"
+          onClick={this.handleDialogClose}
+          disabled={!this.state.changed || this.state.saving}
+        />,
+      ];
+
       return (
         <div style={styles.innerPaper}>
-          <h2>Article Editor: {article.title}</h2>
-          <Divider />
-          <TextField
-            disabled
-            defaultValue={article.title}
-            floatingLabelText="Title"
-            fullWidth
-          />
-          <form
-            onSubmit={this.handleSaveChanges}
+          <Dialog
+            title="Article Editor"
+            actions={actions}
+            open={this.state.open}
+            modal={false}
+            autoScrollBodyContent={true}
+            onRequestClose={this.handleDialogClose}
           >
-            <SelectField
```

Comment:
Only changed the indentation here. not sure why Git created such a mess with the blocks.

Reply:
Yeah I'm afraid git isn't always so smart <.<. It seems like it's taking line 363 as a constant because it used to be an end of React element of Form and now it's an end of React element Dialog, but git thinks that's a line that you "kept"